packages/riipen-ui/src/components/Chip.jsx

```diff
@@ -30,21 +40,41 @@ class Chip extends React.Component {
      */
     disabled: PropTypes.bool,
 
+    /**
+     * Whether to invert colors on hover
+     */
+    hover: PropTypes.bool,
+
+    /**
+     * Icon to display at start of chip
+     */
+    iconStart: PropTypes.elementType,
```

Comment:
Since you can't have an "end" icon, I would be tempted to rename this to just "icon".